frontend/src/components/MapView/utils.ts

```diff
@@ -77,13 +80,16 @@ export const convertToTableData = (result: ExposedPopulationResult) => {
     ? uniq(features.map(f => f.properties && f.properties[key]))
     : [statistic];
 
+  const { adminLevelNames } = getBoundaryLayerSingleton();
+  const lowestLevel = adminLevelNames[adminLevelNames.length - 1];
```

Comment:
I think this appears in a few other places, can you create a util function for this?